BUILDING

```diff
@@ -6,6 +6,33 @@ which includes the Software Development Utilities and Threads options.  If it
 doesn't, this is a bug -- please report it!
 
 
+Tests
+-----
+
+Tests involving AWS require additional setup.
+
+First, you must have AWS access keys.  If you use the web console, this is
+under "My Security Credentials"->"Access keys for CLI, SDK, & API access".  For
+kivaloo, that info should be saved in a text file with the form:
+    ACCESS_KEY_ID=...
+    ACCESS_KEY_SECRET=...
+(with no leading spaces)
+
+- tests/s3: you must:
+  1) have a ~/.s3/aws.key file which contains the access key.
+  2) have a unique S3 bucket name in AWS.
+  3) specify the bucket name as the BUCKET environment variable.  For example,
+
+      $ BUCKET=kivaloo-test-john-smith make test
```

Comment:
Do you want to mention REGION here too?

Reply:
I considered it, but then I figured that the command line would look too ugly.

That said, I guess that we should mention REGION somewhere, so I'll do that.